.github/workflows/continuous-integration-workflow.yml

```diff
@@ -36,6 +36,9 @@ jobs:
           echo "$env:USERPROFILE\.poetry\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
 
       - name: Install Python dependencies
+        if: runner.os == 'Linux'
+        run: |
```

Comment:
It looks the run is defined duplicated, so the action is not triggered.

Reply:
I swear I saw that syntax somewhere, damn. Will fix it soon. Chime in if you know the right way.